firstvoices/backend/models/media.py

```diff
@@ -296,50 +296,37 @@ def get_output_image_size(max_size, input_width, input_height):
     return input_width, input_height
 
 
-class Image(MediaBase):
-    # from fvpicture
-
+class ThumbnailMixin(models.Model):
```

Comment:
Good call on moving the thumbnail fields to a mixin for the Video and Image models.